app/views/idv/session_errors/exception.html.erb

```diff
@@ -12,14 +12,6 @@
           text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
-        decorated_sp_session.sp_name && {
-          url: return_to_sp_failure_to_proof_path(
-            step: 'verify_info',
-            location: request.params[:action],
-          ),
-          text: t('idv.troubleshooting.options.get_help_at_sp', sp_name: decorated_sp_session.sp_name),
-          new_tab: true,
-        },
       ].compact,
```

Comment:
Since there are no `nil`-able values in this array now, we could remove `.compact`.

```suggestion
      ],
```